resources/faq.md

```diff
@@ -245,3 +245,19 @@ In general, we recommend the former. However, if you need to reduce the size of
 **MeiliSearch will never track or identify individual users**. That being said, we do use Amplitude and Sentry to collect anonymous data about user trends and bug reports.
 
 You can read more about what metrics we collect, why we collect them, and how to disable it on our [telemetry page](/learn/what_is_meilisearch/telemetry.md). Transparency is very important to us, so if you feel we are lacking in this area please [open an issue](https://github.com/meilisearch/documentation/issues/new/choose) and let us know! ❤️
+
+## Why does MeiliSearch crash when I try to add documents?
```

Comment:
during indexing, we are using disk too, we could crash if there is no space left on device.

Reply:
Thanks! I added that to the FAQ 👍